packages/fxa-auth-server/lib/routes/session.js

```diff
@@ -627,12 +630,7 @@ module.exports = function (
         const { uid, email } = sessionToken;
         const { code, tokenVerificationId } = options;
 
-        await customs.checkAuthenticated(
-          request,
-          uid,
-          email,
-          'verifySessionCode'
-        );
+        await customs.check(request, email, 'verifySessionCode');
```

Comment:
Should this be checkAuthenticated?

Reply:
I think so... Let me double check this. Thanks for catching.

Reply:
This has been fixed, and tests updated.